perllib/FixMyStreet/Roles/CobrandBulkyWaste.pm

```diff
@@ -463,11 +463,11 @@ sub _bulky_send_reminder_email {
 
 sub bulky_send_before_payment { 0 }
 
-sub bulky_per_item_pricing_by_property_type { 0 }
+sub bulky_per_item_pricing_property_types { [] }
 
 sub bulky_per_item_price_key {
     my $self = shift;
-    return 'price' if !$self->bulky_per_item_pricing_by_property_type;
+    return 'price' if !$self->bulky_per_item_pricing_property_types;
```

Comment:
This will never match, as empty arrayrefs `[]` are not false-y, you'd need `if !@{$self->bulky_per_item_pricing_property_types};` to dereference the arrayref and then check its size.